src/openrct2-ui/windows/Options.cpp

```diff
@@ -681,7 +681,7 @@ class OptionsWindow final : public Window
                 break;
             case WIDX_MINIMIZE_FOCUS_LOSS:
                 gConfigGeneral.minimize_fullscreen_focus_loss ^= 1;
-                platform_refresh_video(false);
+                RefreshVideo(false);
```

Comment:
Weird, is this one out of the namespace or is there an using lurking nearby that I missed?

Reply:
It’s not a platform function, so I dropped that prefix. Same thing applies to the other occurrences.